docs/index.asciidoc

```diff
@@ -762,6 +761,25 @@ It can be either .jks or .p12
 
 Set the keystore password
 
+[id="plugins-{type}s-{plugin}-silence_errors_in_log"]
+===== `silence_errors_in_log`
```

Comment:
doc needs to sort in alphabetical order

Reply:
fixed with 3652086e54e4cd260c50adb3698b5129437b8f13